outputs/core_pools.json

```diff
@@ -11,14 +11,14 @@
     "0x49cbd67651fbabce12d1df18499896ec87bef46f00000000000000000000064a": "sDAI/3Pool",
     "0x596192bb6e41802428ac943d2f1476c1af25cc0e000000000000000000000659": "ezETH-WETH-BPT",
     "0x68e3266c9c8bbd44ad9dca5afbfe629022aee9fe000200000000000000000512": "B-wjAura-wETH",
-    "0x70d5e3234f6329c1d5a26796dcf4e109d69a34880000000000000000000005e7": "uniETH/wstETH/rETH",
```

Comment:
Did we do some pool replacement here?

Reply:
all unieth liquidity appears to be in 0xdd59f89b5b07b7844d72996fc9d83d81acc8219600000000000000000000059e. it is included in the auto generated core pools json